### ISSUE TITLE
add human readable names to kademlia table output

### ISSUE THREAD

**Issue by acud** [ethersphere/swarm#657]
this is a small feature suggestion that i have in the context of simplifying the kademlia table output, in a way that would allow a mere mortal to make (easier) assumptions regarding the state of the hive.
so instead of outputting addresses, we would tuple them with human-readable names that would be unique in the test/simulation level for each address.
in layman terms:
- current output:
```
============ DEPTH: 1 ==========================================
001  1 0518                         |  1 0518 (0)
002  0                              |  0
003  0                              |  0
004  1 52c4                         |  1 52c4 (0)
005  0                              |  0
006  0                              |  0
007  0                              |  0
008  0                              |  0
009  0                              |  0
010  0                              |  0
011  0                              |  0
012  0                              |  0
013  0                              |  0
014  0                              |  0
015  0                              |  0
=========================================================================: 
```

- suggested output:
```
============ DEPTH: 1 ==========================================
001  1 alice                        |  1 alice (0)
002  0                              |  0
003  0                              |  0
004  1 bob                          |  1 bob (0)
005  0                              |  0
006  0                              |  0
007  0                              |  0
008  0                              |  0
009  0                              |  0
010  0                              |  0
011  0                              |  0
012  0                              |  0
013  0                              |  0
014  0                              |  0
015  0                              |  0
=========================================================================: 
```

- alternative suggested output:
```
============ DEPTH: 1 ==========================================
001  1 alice(0518)                  |  1 alice (0)
002  0                              |  0
003  0                              |  0
004  1 bob(52c4)                    |  1 bob (0)
005  0                              |  0
006  0                              |  0
007  0                              |  0
008  0                              |  0
009  0                              |  0
010  0                              |  0
011  0                              |  0
012  0                              |  0
013  0                              |  0
014  0                              |  0
015  0                              |  0
=========================================================================: 
```

**Comment by zelig** [comment]
This is an interesting idea. How would you set the name? via CLI param? it would still fall back to overlay address?

In fact the solution is to finally get rid of this text table, serve it in json where peer struct could contain info like overlay addr, underlay addr, human readable name  and visualisers could display it the way they want
